updatecli/updatecli.d/manifest.yaml

```diff
@@ -123,7 +123,7 @@ targets:
     name: bump core capi in tests package
     kind: file
     spec:
-      file: ./internal/sync/provider_sync_test.go
+      file: ./internal/sync/operator_reconciler_test.go
```

Comment:
I think this file doesn't exist yet.

Reply:
Sorry, true. But it can be added later, in a next PR, as mentioned. Mostly moving parts from the provider_sync_test.go there